pom.xml

```diff
@@ -56,4 +56,56 @@
 			</dependency>
 		</dependencies>
 	</dependencyManagement>
+
+	<profiles>
+		<profile>
+			<id>spring</id>
+			<activation><activeByDefault>true</activeByDefault></activation>
```

Comment:
Looking at the other repos [SCDF](https://github.com/spring-cloud/spring-cloud-dataflow/blob/master/pom.xml#L196) and [SCSt](https://github.com/spring-cloud/spring-cloud-stream/blob/master/pom.xml#L190), I don't see the `spring` profile is activated by default. Do you really needed it to be active in order to successfully import in IntelliJ?